notebooks/QA_evaluation_metrics.ipynb

```diff
@@ -0,0 +1,1271 @@
+{
```

Comment:
<p>Is the F1 score expected to be <code>0.0</code> here?  Same as exact match?</p><p>Or is it that this only makes sense when there are various predictions, where each is compared as exact match and precision/recall/F1 is calculated from the number of exact matches? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>I think so. Since, this is just an example which has single observation. It does makes sense to have F1 score of 0.0. </p>